benchmarks/benchmarks.hpp

```diff
@@ -0,0 +1,34 @@
+//////////////////////////////////////////////////////////////////////////////////////
```

Comment:
This material should not be duplicated against the code in `test/`.  The tests should be added to another file in `tests/`, and specified into different suites.  I can help with that if you need some pointers.

That lets you delete the duplicate catch.hpp, main function, etc.

Reply:
As an example:

`TEST_CASE(“bench_swap_reg”, “[benchmark]”)`, would allow you to specify `benchmark` on the command line, or `~benchmark` (as a default parameter) when not desired.

I don’t think you need the QInterfaceTestFixture either - that can all be self-contained within the `benchmarkLoop` method.